Updated multicut docs (#221)
ilastik/ilastik.github.io: documentation/multicut/multicut.md

## Training and multicut
Now that we have superpixels, we can combine them into objects. If your boundary probability maps are of high quality (e.g. they come from a neural network that was trained on very similar data), you can apply the Multicut partitioning directly on the maps. For this case, leave the "Train edge classifier" checkbox unchecked.
If you are not fully satisfied with your boundary predictions, we can further improve them by training a Random Forest to predict good and bad edges. The general approach we use was first described in [this publication](https://link.springer.com/chapter/10.1007%2F978-3-642-33712-3_56). Briefly, given the superpixels computed in the previous step, we now compute features on the edges of adjacent superpixels.

> These features include the shape of the superpixel boundaries, and intensity statistics computed on the superpixel area/volume as well as along the boundaries.

Reviewer: ```suggestion
These features include the shape of the superpixels, as well as intensity statistics computed along the superpixel boundaries and within the superpixel area/volume.
```

Author: hmmm, so it is really the shape of the boundaries that is considered, not the shape of the superpixels themselves. Do you think that's adding too much detail?

Reviewer: I guess the initial phrasing was slightly confusing to me then 😁 Better?

Author: awww :) almost - would this make sense to you? It's just that it will either be areas (in case of 2D superpixels) or volumes (for 3D supervoxels :))

```suggestion
These features include the shape of the superpixels' boundaries, as well as intensity statistics of the boundaries, and areas (or volumes).
```

Reviewer: Sounds good!

<p>
You can do it in this applet by clicking on the edges between superpixels. Left click, making edge <span style="color: green">green</span>, corresponds to edges which should be <span style="color: green">dropped</span> and right click, making edges <span style="color: red">red</span>, corresponds to true segmentation edges which should be <span style="color: red">preserved</span>. The initial view is shown on the left, while on the right you can see the same image with some edges labeled. To label multiple edges in one sweep, brush across them with either mouse button pressed.
</p>

<a href="snapshots/mc_training_applet_predictions.png" data-toggle="lightbox"><img src="snapshots/mc_training_applet_predictions.png" width="100%" class="img-responsive" alt="Edge predictions from trained edge classifier." /></a>

As usual in ilastik, pressing "Live Predict" will show you the edge probabilities and update them live as you click more edges (left figure).
Now let's apply the multicut and get a consistent segmentation.

#### Multicut

> The multicut algorithm uses boundary evidence to connect superpixels in to larger objects - the multicut segmentation.

Reviewer: ```suggestion
The multicut algorithm uses boundary evidence to connect superpixels into larger objects - the multicut segmentation.
```
